### ISSUE TITLE
Lookup modal is not accessible

### ISSUE THREAD

**Issue by jcoyne** [LD4P/sinopia_editor#2493]
This has many of the same problems as the accessible modal:
See #2450, #2451, #2454, #2455

**Comment by justinlittman** [comment]
No longer a modal.
